### PR TITLE
Add GHA for SubM Tweeting via PRs

### PR DESCRIPTION
**Pull request by dfarrell07**
Add a GitHub action that will enable the community to manage
Submariner's Twitter by PRs to .tweet files.

This can be used for announcing releases, as Submariner's Twitter is
currently mostly used for, but can also be used for other Submariner
content. For example, to highlight media about Submariner or events
Submariner will take part in. In general, this opens up Submariner's
Twitter to the community, subject to the community's code review.

This also increases Submariner's "bus factor", which is a requirement
for CII Silver.

Signed-off-by: Daniel Farrell <dfarrell@redhat.com>

### CODE REVIEW

**Comment by nyechiel** [comment]
@dfarrell07 can you clarify what this is intended to be used for? Is it for announcing releases?

**Comment by dfarrell07** [comment]
> @dfarrell07 can you clarify what this is intended to be used for? Is it for announcing releases?

@nyechiel Addressed in the commit message, thanks!

**Comment by dfarrell07** [comment]
I thought about adding this to the submariner-website repo vs the main repo, don't have a strong opinion.

**Comment by dfarrell07** [comment]
> > @dfarrell07 can you clarify what this is intended to be used for? Is it for announcing releases?
> 
> @nyechiel Addressed in the commit message, thanks!

Also added:

> This also increases Submariner's "bus factor", which is a requirement
for CII Silver.

**Comment by nyechiel** [comment]
@sridhargaddam @tpantelis @mkolesnik @skitt can you please review?

**Review by tpantelis — APPROVED** [review]

File: .github/workflows/twitter-together.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  tweet:
+    name: Tweet
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push'
+    steps:
+      - name: Check out the repository
+        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
+
+      - name: Tweet
+      - uses: gr2m/twitter-together@f72bdb719fed3f924ecd711086181d66af4eb72e
```
Comment (dfarrell07) [inline .github/workflows/twitter-together.yml]:
```suggestion
      uses: gr2m/twitter-together@f72bdb719fed3f924ecd711086181d66af4eb72e
```

About to send an update to fix...

**Review by skitt — APPROVED** [review]

**Pull request merged**